pkg/config/remote/client.go

```diff
@@ -251,20 +251,29 @@ func (c *Client) startFn() {
 // pollLoop should never be called manually and only be called via the client's `sync.Once`
 // structure in startFn.
 func (c *Client) pollLoop() {
+	firstRun := true
 	for {
 		interval := c.backoffPolicy.GetBackoffDuration(c.backoffErrorCount)
 		select {
 		case <-c.ctx.Done():
 			return
 		case <-time.After(c.pollInterval + interval):
-			c.lastUpdateError = c.update()
-			if c.lastUpdateError != nil {
-				c.backoffPolicy.IncError(c.backoffErrorCount)
-				log.Errorf("could not update remote-config state: %v", c.lastUpdateError)
+			err := c.update()
+			if err != nil {
+				if firstRun {
+					// As some clients may start before the core-agent server is up, we log the first error
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Let's add a JIRA card to remove this part, and make sure to have the future Remote Config backend client component as a dependency of the RC client component

Reply:
Created [RCM-1086](https://datadoghq.atlassian.net/browse/RCM-1086) for this 👍